resume/apps.py

```diff
@@ -2,4 +2,4 @@
 
 
 class ResumeConfig(AppConfig):
-    name = "Resume"
+    name = "resume"
```

Comment:
Failed in Django 3 because it tried to import a module `Resume` that does not exist.